.utility/push-docker.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e  # Exit on error
+
+echo "Welcome to loklak Docker hub push script"
+
+if [ "$TRAVIS_REPO_SLUG" != "loklak/loklak_server" ]; then
+    echo "Skipping Docker push for repo $TRAVIS_REPO_SLUG"
+    exit 0
+fi
+
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+    echo "Skipping Docker push for pull request"
```

Comment:
The logic is this

If `$TRAVIS_PULL_REQUEST` is not false

i.e. if `$TRAVIS_PULL_REQUEST` is true

i.e. if current build is for a pull request

&nbsp;&nbsp;&nbsp;&nbsp;then print that skipping push for this pull request